# Goal
float32 target_position
---
# Result
bool success
---
# Feedback
float32 current_position
